src/pages/workspace/WorkspaceInitialPage.js

```diff
@@ -80,6 +82,18 @@ class WorkspaceInitialPage extends React.Component {
         Navigation.navigate(ROUTES.SETTINGS);
     }
 
+    /**
+     * @returns {Boolean}
+     */
+    hasPolicyCreationError() {
+        return Boolean(this.props.policy.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD && this.props.policy.errors);
```

Comment:
The above logic caused issue:
- https://github.com/Expensify/App/issues/41325

because optimistic / success data is setting `requiresCategory` to `null`, therefore `policy.errors` will always be `true`.

https://github.com/Expensify/App/blob/942c851a6bcdb4d41df61a97bc097e14424f1017/src/libs/actions/Policy.ts#L3571-L3599

The proposed and implemented solution to fix this was to check `!isEmptyObject(policy.errors)` instead.